pkg/services/network/constants.go

```diff
@@ -22,6 +22,10 @@ const (
 	NSXTTypeNetwork = "nsx-t"
 	// NSXTVNetSelectorKey is also defined in VM Operator.
 	NSXTVNetSelectorKey = "ncp.vmware.com/virtual-network-name"
+	// NSXVPCTypeNetwork is the name of the NSX VPC network type.
+	NSXVPCTypeNetwork = "nsx-t-subnetset"
+	// NSXVPCSubnetSetSelectorKey has to be aligned with VM Operator design.
+	NSXVPCSubnetSetSelectorKey = "nsx-operator.vmware.com/subnetset-name"
```

Comment:
Is this some key coming from the nsx-operator? Can we add a reference link to it if so?
What is it used for?

We should align the naming to the decision above if possible.

Reply:
These are the annotation keys designed for the VirtualMachineService and LoadBalancer type service, originally defined in CAPW. It's quite uncertain if vm-operator/ncp/nsx-operator utilize them anymore. We should contemplate removing all of them once their usage in vm-operator/ncp/nsx-operator has been confirmed and the adjustment could be undertaken in a separate, dedicated fix.